apiary.apib

```diff
@@ -2098,7 +2121,7 @@ communication with DeepAffex. When set, the server will compare the token value
 with HTTP request's `Referer` header and reject the request if those don't
 match.
 
-**Note:** Please make sure to store the obtained token locally as most
+**Note:** Please make sure to store the obtained tokens locally as most
```

Comment:
I think we should replace the word `locally` with `securely` - the storage doesn't need to be local but it does need to be secure.